### PR TITLE
[DO NOT MERGE] Update requirements for Scrapy 2.10.0.

### PR DESCRIPTION
**Pull request by wRAR**

### CODE REVIEW

**Comment by wRAR** [comment]
Ping @Gallaecio if any adjustments are needed

File: requirements.in [hunk 1]
```diff
@@ -34,7 +34,6 @@ boto3
 # address known vulnerabilities
 aiohttp>=3.7.4      # CVE-2021-21330
 awscli>=1.19.0      # fix pyyaml dependency
-cryptography>=3.3.2 # CVE-2020-36242
```
Comment (wRAR) [inline requirements.in:37]: Scrapy requires 36.0.0+

**Review by Gallaecio — COMMENTED** [review]
That Scrapy==2.9.0  on top seems problematic :)

I wonder if we need versions at all in requirements.in (except for `<` rules, which I see none), as long as we regenerate requirements.txt from scratch (i.e. remove it, then run `pip-compile`) to force the latest versions of everything.

**Comment by wRAR** [comment]
> That Scrapy==2.9.0 on top seems problematic :)

I wonder what did I change then.

**Comment by VMRuiz** [comment]
Shouldn't we push a new branch instead of merging over branch 2.9?

**Comment by wRAR** [comment]
You can't push a new branch via a Github PR :(

**Comment by Gallaecio** [comment]
He does have a point that we should point elsewhere, I assume we don’t want to merge these changes into branch-2.9.

**Comment by wRAR** [comment]
Previous times these PRs were not merged at all, I think the branches were used directly. But that's up to @elacuesta

**Review by Gallaecio — APPROVED** [review]

**Comment by wRAR** [comment]
Make sure that Twisted isn't bumped to 23.8.0 when finalizing the stack changes.

**Comment by elacuesta** [comment]
Tags `2.10`, `2.10-latest` & `2.10-20230901` available in dockerhub: https://hub.docker.com/r/scrapinghub/scrapinghub-stack-scrapy/tags?page=1&name=2.10. Thanks everyone!

**Pull request closed without merging**